docs/Architecture/7_dev_setup.md

```diff
@@ -0,0 +1,63 @@
+# 🤖 Developer Setup
+
+## **Prerequisites**
+
+Before you begin, ensure you have the following tools installed:
+
+-   [Git](https://git-scm.com/downloads)
+-   [Go](https://go.dev/doc/install)
+-   [Make](https://www.gnu.org/software/make)
+-   [Mise](https://mise.jdx.dev/getting-started.html)
+-   [Temporal](https://docs.temporal.io/cli)
+-   [Tmux](https://github.com/tmux/tmux/wiki/Installing)
+
+## **Installation Steps**
+
+### 1. Install `slangroom-exec`
+
+Download the appropriate `slangroom-exec` binary for your OS from the [releases page](https://github.com/dyne/slangroom-exec/releases).
+
+Add `slangroom-exec` to PATH and make it executable:
+
+```bash
+wget https://github.com/dyne/slangroom-exec/releases/latest/download/slangroom-exec-Linux-x86_64 -O slangroom-exec
+chmod +x slangroom-exec
+sudo cp slangroom-exec /usr/local/bin/
+```
+
+### 2. Install `zencode-exec`
+
+Download the appropriate `zencode-exec` binary for your OS from the [releases page](https://github.com/dyne/zenroom/releases).
+
+> [!WARNING]
+> On Mac OS, the executable is zencode-exec.command and you have to symlink it to zencode-exec
+
+Add `zencode-exec` to PATH and make it executable:
+
+```bash
+wget https://github.com/dyne/zenroom/releases/latest/download/zencode-exec
+chmod +x zencode-exec
+sudo cp zencode-exec /usr/local/bin/
+```
+
```

Comment:
zencode-exec also we do not need since we rely on slangroom-exec that has the same functionalities with some extra <3